index.js

```diff
@@ -227,7 +227,13 @@ module.exports = function(bundle, opts) {
   }
 
   function fileKey(filename) {
-    return path.relative(basedir, filename);
+    const relative = path.relative(basedir, filename);
+    if (/^[a-zA-Z0-9]/.test(relative)) {
+      // Example: return ./Label.jsx instead of Label.jsx - reverse module id
+      // lookup would not work with just Label.jsx
+      return './' + relative
+    }
+    return relative
```

Comment:
Is this change still necessary with the initModule change? I can see how this change by itself could solve the issue in the specific case with multiple source files each in the basedir, but I think the initModule change alone should solve the general case.

Reply:
You're right, I do not think this change is necessary. I had thought it wasn't at first, but when I tested it, it didn't work. I've just tested it again with the original impl of `fileKey` and it seems to work (no error from `react-hot-loader`).